cypress/integration/App.spec.js

```diff
@@ -0,0 +1,55 @@
+describe('App Test', () => {
+  it('Visit the app', () => {
+    // first visit the app
+    cy.visit('/');
+
+    // check if we have tha class container
+    cy.get('.container').should('be.visible');
+
+    // check if in the DOM there exist the class called app-title and has text Home
+    cy.get('.app-title').should('exist').and('has.text', 'Home');
+  });
+
+  it('Visit the Sign Up in dummy component', () => {
+    // first visit the app
+    cy.visit('/sign-up');
+    // check if we have tha class container
+    cy.get('.container').should('be.visible');
+
+    // check if we have tha class container
+    cy.get('.container').should('be.visible');
+
+    // check if in the DOM there exist the class called app-title and has text Sign up
+    cy.get('.app-title').should('exist').and('has.text', 'Sign up');
+  });
+
+  it('Visit the About in dummy component', () => {
+    // first visit the app
+    cy.visit('/about');
+    // check if we have tha class container
+    cy.get('.container').should('be.visible');
+
+    // check if we have tha class container
+    cy.get('.container').should('be.visible');
+
+    // check if in the DOM there exist the class called app-title and has text About
+    cy.get('.app-title').should('exist').and('has.text', 'About');
+  });
+
+  it('Visit the Welcome Page', () => {
+    // first visit the app
+    cy.visit('/welcome');
+
+    // check if we have tha class container
+    cy.get('.container').should('be.visible');
+
+    // check if we have tha class container
+    cy.get('.container').should('be.visible');
+
+    // check if in the DOM there exist the class called app-title and has text Authors Haven
+    cy.get('.app-title').should('exist').and('has.text', 'Authors Haven');
+
+    // check if we are getting the actual message from the API call
+    cy.get('.container').should('contain', 'Welcome to Authors Haven');
```

Comment:
This test is failing in my local machine? Any insights why?

Reply:
OOhhhhhh @avpaul gave you the wrong `REACT_APP_API` value.  Based to the code I saw from Shema's PR. This is the value user in the .env file `https://ah-kg-avengers-backend-staging.herokuapp.com/`. Remove api/v1

Reply:
Thank you @niyobobo I think that is the issue.

Reply:
Checked on it and its runs @niyobobo and @shemaeric. 
What about the CircleCI?